### PR TITLE
Python Examples

### PR DESCRIPTION
**Pull request by pavankumarbn**
@lorensen @ajpmaclean Please review these Python Examples.

### CODE REVIEW

File: src/Python/Filtering/TriangulateTerrainMap.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+def main():
+
+    colors = vtk.vtkNamedColors()
+    # Create points on an XY grid with random Z coordinate
+    points = vtk.vtkPoints()
+    
+    for x in range (0, 10):
+        for y in range (0, 10): 
+           points.InsertNextPoint(x, y, random.uniform(0.0, 3.0))
+
```
Comment (ajpmaclean) [inline src/Python/Filtering/TriangulateTerrainMap.py]: I suggest removing random and using `vtkMinimalStandardRandomSequence` instead. Look at [DelaunayMesh.py](https://lorensen.github.io/VTKExamples/site/Python/Modelling/DelaunayMesh/) for how to use it.

Comment (pavankumarbn) [reply src/Python/Filtering/TriangulateTerrainMap.py]: @ajpmaclean Thank You! I have made the changes as suggested. Please review on the same.

Comment (pavankumarbn) [reply src/Python/Filtering/TriangulateTerrainMap.py]: @ajpmaclean Sure! I will do it

Comment (pavankumarbn) [reply src/Python/Filtering/TriangulateTerrainMap.py]: @ajpmaclean I have updated. Please review on the same.

**Comment by ajpmaclean** [comment]
Thanks

**Comment by pavankumarbn** [comment]
@ajpmaclean These examples are not listed on the website. After the website sync are they going to appear?

**Comment by ajpmaclean** [comment]
They are have been merged and will appear when the web site is updated. At present there is a major re-write going on. It is good to see you are verifying how they look on the web site ... a good way to pick up any problems.

**Comment by pavankumarbn** [comment]
@ajpmaclean Thanks for the update. Yes, Sometimes there would be some changes to give nice look from your side. That helps me to incorporate those changes in next examples.

**Comment by lorensen** [comment]
I synced today.

**Comment by pavankumarbn** [comment]
@ajpmaclean Sure! I will check. Thank you

**Pull request merged**